### ISSUE TITLE
Problem In TreeHouse Downloader

### ISSUE THREAD

**Issue by dordejotri12** [gebeto/python-projects#1]
Hi bro!
The project of Treehouse Downloader isn't working yet...
I tested it a lot but it show thtat there's an errors in the lines of code
please, fix it

**Comment by gebeto** [comment]
>Hi bro! ...

@dordejotri12 If there are problem in the code, maby you neet to use another version of the python

**Comment by dordejotri12** [comment]
> > Hi bro! ...
> 
> @dordejotri12 If there are problem in the code, maby you neet to use another version of the python

I use the last version of Python

**Comment by gebeto** [comment]
>> > Hi bro! ... ...

@dordejotri12 I wrote it a long time ago on python 2, use python 2 to run it

**Comment by gebeto** [comment]
Maby the script not working now, because it is realy old

**Comment by dordejotri12** [comment]
> Maby the script not working now, because it is realy old

may it stills work because when i run the script it tells me that there's a problem in the lines of code
look at the problem please!
https://ibb.co/P6M9W9v

**Comment by dordejotri12** [comment]
Is there another script for downloading full courses from TreeHouse because It's tired to download the course by video...

**Comment by gebeto** [comment]
>Is there another script for downloading full courses from TreeHouse because It's tired to download the course by video...

@dordejotri12 yeah, it is problem with syntax of PRINT statement, just use python 2

**Comment by dordejotri12** [comment]
> > Is there another script for downloading full courses from TreeHouse because It's tired to download the course by video...
> 
> @dordejotri12 yeah, it is problem with syntax of PRINT statement, just use python 2.7

i used python 2.7 but unfortunately :( , another problem appears
https://i.ibb.co/8XwXXWT/problem.png

**Comment by gebeto** [comment]
>> > Is there another script for downloading full courses from TreeHouse because It's tired to download the course by video... ...

@dordejotri12 you need to install all dependencies
"requests" is one of that

**Comment by dordejotri12** [comment]
yes i already know the problem but my problem is that i can't install pip on python 2.7.0 because i don't find a version suitable with python's version

**Comment by dordejotri12** [comment]
look please
https://i.ibb.co/10Kd1jr/problem.png

**Comment by dordejotri12** [comment]
Bro look at this i've done all of things, i installed pip + requests module and i run the script but look at this please:
https://i.ibb.co/RQgrVbj/problem.png
where is the problem here ?
